test/ConnectionManager.tests.js

```diff
@@ -88,6 +88,7 @@ define(function (require) {
                 expect(mockConnectionFactory.createConnection).toHaveBeenCalled();
                 expect(mockConnection.execute).toHaveBeenCalledWith(
                     jasmine.any(Request),
+                    jasmine.any(Object),
```

Comment:
any reason to fix that only in line 91 and not in others similar cases line 121, 148, 175, 203 and here ?